### ISSUE TITLE
Inject a "comment" action into shortcuts downloaded from the service

### ISSUE THREAD

**Issue by insidegui** [insidegui/Sharecuts#20]
Maybe we could inject a comment action to the beginning of every shortcut downloaded from the service with a permalink to the shortcut, so that it's easier to keep track of where it came from.

**Comment by esaruoho** [comment]
Absolutely, this would be extremely helpful and would drive adoption. Also, it would also be incredibly useful if we could attach animated GIFs of the workflows in action - and if possible, also, a screenshot of the contents of the Shortcut - so people would feel less scared of installing them.

**Comment by insidegui** [comment]
@esaruoho That's planned in #21
